ml/v1/sizes.go

```diff
@@ -0,0 +1,33 @@
+//
+// DISCLAIMER
+//
+// Copyright 2023 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+package v1
+
+const (
+	// SizeSmall specifies the Size ID for small MLServices size.
+	// 4 vCPU / 16GB Memory
+	SizeSmall = "small"
+	// SizeMedium specifies the Size ID for medium MLServices size.
+	// 8 vCPU / 32GB Memory
+	SizeMedium = "medium"
+	// SizeLarge specifies the Size ID for large MLServices size.
+	// 16 vCPU / 64GB Memory
+	SizeLarge = "large"
+)
```

Comment:
please remove, shouldn't be needed

Reply:
I added it mostly so that we can use the `small` const in the code for setting a default. However, shouldn't be needed anymore with the `is_default field`. Removed